documentation/source/release_notes.rst

```diff
@@ -0,0 +1,50 @@
+#############
+Release Notes
+#############
+
+All releases are available from the `GitHub Releases Page <https://github.com/potentialventures/cocotb/releases>`_.
+
+cocotb 1.2
+==========
+
+Released on DD MM YYYY.
+
+New features
+------------
+
+- cocotb is now built as Python package and installable through pip. (:pr:`517`, :pr:`799`, :pr:`800`, :pr:`803`, :pr:`805`)
+- Support for async functions and generators was added (Python 3 only). Please have a look at https://cocotb.readthedocs.io/en/latest/coroutines.html#async-functions for an example how to use this new feature.
```

Comment:
You should be able to use a relative link here, so that it works on a local build too. I don't remember the syntax though.

Reply:
Good point, I've updated the link to use an internal reference.